### ISSUE TITLE
MiKo_3081 shall not apply to boolean flag switches

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#729]
MiKo_3081 should not report a boolean flag switch as in following example:
```c#
        someFlag = !someFlag;
```
